src-java/floodlightrouter-topology/floodlightrouter-storm-topology/src/main/java/org/openkilda/wfm/topology/floodlightrouter/model/RegionMappingUpdate.java

```diff
@@ -13,14 +13,15 @@
  *   limitations under the License.
  */
 
-package org.openkilda.wfm.topology.floodlightrouter.service;
+package org.openkilda.wfm.topology.floodlightrouter.model;
 
 import org.openkilda.model.SwitchId;
 
 import lombok.Value;
 
 @Value
-public class SwitchMapping {
+public class RegionMappingUpdate {
     private SwitchId switchId;
     private String region;
+    private boolean readWriteMode;
```

Comment:
The same naming issue. Rename to writeAllowedMode.

Reply:
+1

Reply:
Same reason to keep this name.